### ISSUE TITLE
new Buffer is deprecated

### ISSUE THREAD

**Issue by joshunger** [spotify/web-api-auth-examples#24]
new Buffer is deprecated.

See https://nodejs.org/api/buffer.html.

Usages: https://github.com/spotify/web-api-auth-examples/search?utf8=%E2%9C%93&q=buffer&type=

**Comment by arirawr** [comment]
Good catch! @joshunger would you be interested in making a PR for this? 😄

**Comment by joshunger** [comment]
Thanks @zzwong !
